test-utils/sls-test-utils.ts

```diff
@@ -0,0 +1,59 @@
+import { type Template } from 'cloudform'
+import pino from 'pino'
+
+// Serverless Framework provides plugins with a logger to use, so we simulate that with this:
+export const dummyLogger = Object.assign({}, pino())
+const extras = ['levels', 'silent', 'onChild', 'trace', 'debug', 'info', 'warn', 'error', 'fatal']
+for (const extra of extras) {
+  // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
```

Comment:
I know this code was only moved but would this work better 

```typescript
const extras = ['levels', 'silent', 'onChild', 'trace', 'debug', 'info', 'warn', 'error', 'fatal']
const p = pino()
export const dummyLogger = Object.keys(p).filter(key => !extras.includes(key))
    .reduce((cur, next)=> ({...cur, ...p[next]}), {});
```

Reply:
Great suggestion. I'd like to avoid the `reduce` if possible, since it takes me a bit more time to understand it.
How about this?

```typescript
const extras = ['levels', 'silent', 'onChild', 'trace', 'debug', 'info', 'warn', 'error', 'fatal']
const pinoLogger = pino()
export const dummyLogger = Object.fromEntries(
  Object.entries(pinoLogger).filter(([key]) => !extras.includes(key))
)
```